gym/lib/gym/xcodebuild_fixes/generic_archive_fix.rb

```diff
@@ -0,0 +1,32 @@
+require 'shellwords'
+
+module Gym
+  class XcodebuildFixes
+    class << self
+      # Determine IPAs for the Watch App which aren't inside of a containing
+      # iOS App and removes them.
+      #
+      # In the future it may be nice to modify the plist file for the archive
+      # itself so that it points to the correct IPA as well.
+      #
+      # This is a workaround for this bug
+      # https://github.com/CocoaPods/CocoaPods/issues/4178
+      def generic_archive_fix
+        UI.verbose "Looking For Orphaned WatchKit2 Applications"
+
+        Dir.glob("#{BuildCommandGenerator.archive_path}/Products/Applications/*.app").each do |app_path|
+          info_plist_path = "#{app_path}/info.plist"
```

Comment:
Our belief is that `Info.plist` should be capitalized. Is there any reason that isn't/can't be so?